sales.js

```diff
@@ -13,14 +13,15 @@ for(var i = 0; i < allLocations.length; i++){
   var newList = document.createElement('ul');
   document.body.appendChild(newList);
 
-//If we copy the model above could we take simulatecustomersPerHour, use a for loop and store the lists of cookies per hours as an array called customersPerHourPerLocation.
+//If we copy the model above could we take simulatecustomersPerHour, use a for loop and store the lists of cookies per hour as an array called customersPerHourPerLocation.
   for(var j = 0; j < currentLocation.customersPerHour.length; j++){
     var customerCount = currentLocation.customersPerHour[j];
-    var time = (j + 6);
-    console.log('hours: ' + convert24HourTime(time) + ', cookies: ' + customerCount);
-
+    var time = convert24HourTime(j + 6);
+    var timeAndCookies = ('Time: ' + (time) + ', Cookies Sold: ' + customerCount.toFixed(0));
   // then we could append the stored arrays and insert them under the h3 element Create a unorderd list of the time followed by the number of cookies sold.
-
+    var cookiesPerHour = document.createElement('li');
+    cookiesPerHour.textContent = timeAndCookies;
+    document.body.appendChild(cookiesPerHour);
```

Comment:
Check the location of your `<li>`s in the Elements tab of the Inspector. I don't think they're being appended to the correct parent.